src/tests/unit/tests/electron/platform/android/rule-information-provider.test.ts

```diff
@@ -249,4 +260,22 @@ describe('RuleInformationProvider', () => {
             expect(ruleInformation.getResultStatus(ruleResult)).toBe(testCase.outcome);
         },
     );
+
+    it.each([
+        ['available when ATFA flag is disabled', false],
+        ['not available when ATFA flag is enabled', false],
```

Comment:
Should this be `true` for enabled feature flag?

Reply:
Durp! Fixed.